pkgs/development/interpreters/erlang/generic-builder.nix

```diff
@@ -174,24 +161,9 @@ stdenv.mkDerivation (
           --replace-fail '"sh ' '"${runtimeShell} '
       '';
 
-    # For OTP 27+ we need ex_doc to build the documentation
-    # When ex_docSupport is enabled, grab the raw ex_doc executable from the ex_doc
-    # derivation. Next, patch the first line to use the escript that will be
-    # built during the build phase of this derivation. Finally, building the
-    # documentation requires the erlang-logo.png asset.
-    preConfigure =
-      ''
-        ./otp_build autoconf
-      ''
-      + optionalString ex_docSupport ''
-        mkdir -p $out/bin
-        cp ${ex_doc}/bin/.ex_doc-wrapped $out/bin/ex_doc
-        sed -i "1 s:^.*$:#!$out/bin/escript:" $out/bin/ex_doc
-        export EX_DOC=$out/bin/ex_doc
-
-        mkdir -p $out/lib/erlang/system/doc/assets
-        cp $src/system/doc/assets/erlang-logo.png $out/lib/erlang/system/doc/assets
-      '';
+    preConfigure = ''
+      ./otp_build autoconf
```

Comment:
Since about Erlang/OTP 24 you no longer have to run autoconf, unless the nix build patches any of the configure.ac files.

Reply:
We're not patching at all and both 25, 27 build without autoconf so I've removed it. Thanks. :)